client/my-sites/stats/annual-site-stats/index.js

```diff
@@ -89,19 +127,25 @@ class AnnualSiteStats extends Component {
 		const previousYearData = previousYear && years && find( years, y => y.year === previousYear );
 		const isLoading = ! years;
 		const isError = ! isLoading && years.errors;
-		const noData = ! isLoading && ! isError && ! ( currentYearData || previousYearData );
+		const hasData = isWidget ? currentYearData || previousYearData : years;
+		const noData = ! isLoading && ! isError && ! hasData;
+		const noDataMsg = isWidget
+			? translate( 'No annual stats recorded for this year' )
+			: translate( 'No annual stats recorded' );
```

Comment:
Hi! I've found a possible matching string that has already been translated **19** times:
`translate( 'No annual stats recorded for this year' )` *ES Score: 12*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).